x/evm/types/params.go

```diff
@@ -66,8 +80,16 @@ func (p Params) Validate() error {
 func validateEVMDenom(i interface{}) error {
 	denom, ok := i.(string)
```

Comment:
this cheks and passes if the passed denom is a string but do need to check if it's the same as the one set in genesis?

Reply:
this is just local validation. We don't have access to the SDK context unfortunately from the parameters in order to test against other variables